scripts/qa_crowbarsetup.sh

```diff
@@ -98,6 +98,8 @@ export want_postgresql=${want_postgresql:-1}
 if iscloudver 6plus ; then
     export CROWBAR_EXPERIMENTAL=true
     export CROWBAR_VERIFY_SSL=false
+    # set timeout to 10min to avoid bsc#988030
+    export CROWBAR_TIMEOUT=600
```

Comment:
> before the change [...]

right, that's what we know now - we didn't knew about the mess before
So its good that we found this during this discussion.

> And why are you saying that raising to 1 hour as I suggested would not solve the problem?

I didn't say that. I even proposed the same :)  But I would also make it consistent.

Let me repeat the history for you:

- We saw mkcloud builds of SAP PRs failing in crowbar commit due to timeouts.
- We could not change the timeout for the crowbar command
- As the crowbar command is obsolete anyway, we had the idea to switch to crowbarctl (for now only for proposal commit), as we can change the timeout from mkcloud via env variable
- then this PR was created
- and then during the discussion we found out about the configuration mess